packages/client/src/utils/buttonActions.js

```diff
@@ -327,6 +327,11 @@ const showNotificationHandler = action => {
   notificationStore.actions[type]?.(message, autoDismiss)
 }
 
+const promptUserHandler = action => {
```

Comment:
We can just make this a completely empty function:
`const promptUserHandler = () => {}`

Doesn't affect functionality of course but may as well keep it as tidy as possible!